pkg/backend/openshiftcluster.go

```diff
@@ -105,8 +116,15 @@ func (ocb *openShiftClusterBackend) handle(ctx context.Context, log *logrus.Entr
 			log.Error(err)
 			return ocb.endLease(ctx, stop, doc, api.ProvisioningStateFailed)
 		}
-
-		return ocb.endLease(ctx, stop, doc, api.ProvisioningStateSucceeded)
+		doc, err = ocb.db.OpenShiftClusters.Get(ctx, doc.ID)
+		if err != nil {
+			log.Error(err)
+			return ocb.endLease(ctx, stop, doc, api.ProvisioningStateFailed)
+		}
+		if doc.OpenShiftCluster.Properties.Install == nil {
+			return ocb.endLease(ctx, stop, doc, api.ProvisioningStateSucceeded)
+		}
+		return ocb.endLease(ctx, stop, doc, api.ProvisioningStateUpdating)
```

Comment:
Sorry, I have provided a bad steer here.  Update is not a multi-phase process and Install is always nil by this point.  This hunk should be reverted.